make-xges.py

```diff
@@ -27,11 +27,57 @@ def file_to_uri(path):
     path = os.path.realpath(path)
     return 'file://' + path
 
+def my_time_format(raw_time):
```

Comment:
Maybe call this `parse_time` instead?

Reply:
Is that an external dependency?

Reply:
No.  I was suggesting a different name for the function that better describes what it does.

Reply:
OK.